### DISCUSSION TITLE
Volume control keys controlling output levels of multiple devices?

### DISCUSSION THREAD

**Discussion by jflyper** [ExistentialAudio/BlackHole#675] [Q&A]
Hi. I'm fiddling with an audio routing setup for my M1PRO macbook, and scratching my head for days now.

The problem is that my volume control key is simultaneously controlling output levels of multiple devices. For example, if I have an output path: System Sound -> Background -> Blackhole 2ch -> Macbook Pro Speaker, then my volume control keys will alter output levels of Background AND Blackhole 2ch.

Is this behavior correct and natural to other users? My gut instinct tells me the keys should control level of final output device only.

**Comment by devinroth** [comment]
What’s Background? Background Music Driver?

**Reply by jflyper** [reply]
Ah, sorry, Background is _Background Music_ utility:
https://github.com/kyleneideck/BackgroundMusic

**Comment by devinroth** [comment]
Volume keys only work for the default audio device. I suspect Background Music is doing something. This is a question for Them.
